packages/edit-widgets/src/components/header/style.scss

```diff
@@ -45,8 +45,9 @@
 
 	// The Toolbar component adds different styles to buttons, so we reset them
 	// here to the original button styles
-	> .components-button.has-icon,
-	> .components-dropdown > .components-button.has-icon {
+	// Specificity bump needed to offset https://github.com/WordPress/gutenberg/blob/8ea29cb04412c80c9adf7c1db0e816d6a0ac1232/packages/components/src/toolbar/style.scss#L76
+	> .components-button.has-icon.has-icon.has-icon,
```

Comment:
Is it necessary to add it 3 times?

Reply:
The rule we're overriding has 4 classnames, so I thought it would be best to have 5 to be on the safe side 😅